src/editor/Collaboration.js

```diff
@@ -0,0 +1,70 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var SimpleWebRTC = require("simplewebrtc");
+
+    function Collaboration() {
+        var self = this;
+        var webrtc = new SimpleWebRTC({
+          // the id/element dom element that will hold "our" videos
+          localVideoEl: 'localVideo',
+          // the id/element dom element that will hold remote videos
+          remoteVideosEl: 'remotesVideos',
+          // immediately ask for camera access
+          autoRequestMedia: true
+        });
+        this.webrtc = webrtc;
+        this.webrtc.on('readyToCall', function () {
+          webrtc.joinRoom('thimble');
+        });
+        this.init();
+    };
+
+    Collaboration.prototype.init = function() {
+        var self = this;
+        self.changing  = false;
+        this.webrtc.connection.on('message', function (msg) {
+          if(msg.type === "data") {
```

Comment:
Invert this check, and return early if it fails, so you don't have to indent all the code below so much:

```js
if(msg.type !== "data") {
  return;
}
...
```